code/modules/sanity/sanity_mob.dm

```diff
@@ -223,6 +223,8 @@
 	print_desires()
 
 /datum/sanity/proc/print_desires()
+	if(owner.stats.getPerk(PERK_ARTIST))
+		to_chat(owner, SPAN_NOTICE("You desire [english_list(desires)]."))
 	if(!resting)
 		return
 	to_chat(owner, SPAN_NOTICE("You desire [english_list(desires)]."))
```

Comment:
Bruh.

Reply:
necessary due to lack of resting on artists